src/components/Form/FormFormats.js

```diff
@@ -1,46 +1,49 @@
 /* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable func-name-matching */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable newline-per-chained-call */
+/* eslint-disable array-element-newline */
+/* eslint-disable sort-keys */
+/* eslint-disable object-property-newline */
 import * as types from "actions/actionTypes";
 import {instance} from "actions/apiActions";
-import {useSelector, useDispatch} from "react-redux";
+import {useSelector} from "react-redux";
 import React from "react";
 import {FORM_ERROR} from "final-form";
 import * as Fields from "./Fields";
 import * as Yup from "yup";
 import * as hooks from "actions/hooks";
+import * as moment from 'moment';
 
-const shortToLongGender = {
-    "F": "female",
-    "M": "male",
-    "U": "unspecified",
-};
-const longToShortGender = {
-    "female": "F",
-    "male": "M",
-    "unspecified": "U",
-};
-
-const mapFunc = (parser, data) => {
-    let res = {};
+export const responseToForm = (parser, data) => {
```

Comment:
What's parser? consider using a more descriptive variable name

Reply:
just to clarify, is this ApiResponseToFormFormat ?

Reply:
This won't be used anymore, it was supposed to map response errors to form fields.